src/i18n/en.js

```diff
@@ -487,13 +487,22 @@ deletion, security, cross-border data transfers and other issues.
     },
   },
   subscriber_gdpr: {
-    description: 'Here you can find all permissions given by your Subscriber.',
+    consent: 'Consent',
+    description: 'Here you will find the history of all consents given by your Subscriber.',
     empty_data: 'This Subscriber has not given or denied any permission.',
     empty_html_text: 'With no legal text defined',
+    latest_results: 'Last 10 results',
+    modification_date: 'Modification date',
+    modification_source_ip: 'Modification source ip',
+    origin_type: {
+      form: 'Form',
+      manually: 'Manually',
+    },
```

Comment:
Where is it being used?

BTW, there are more origins than this:

* manual,
* import,
* form,
* mercadoshop,
* api,
* tokkoBroker,
* tiendanube,
* datahub,
* vtex,
* shopify,
* magento,
* wordpress,
* unbounce,
* prestashop,
* woocommerce,
* learnpress,
* crm2go,
* zoho,
* typeform,
* piesync,
* dynamics365

Reply:
In the double we only have this two types [here](https://github.com/FromDoppler/doppler-webapp/blob/master/src/services/doppler-api-client.double.ts#L474)

Reply:
These are only examples.

Reply:
Removed. Resolving conversation...